app/javascript/components/FacetsPanel.js

```diff
@@ -1,20 +1,20 @@
 import React, { useState, useEffect, useContext } from 'react';
 import FacetControl from './FacetControl';
 import MoreFacetsButton from './MoreFacetsButton';
-import { StudySearchContext } from 'components/search/StudySearchProvider'
+import { SearchFacetContext } from 'components/search/SearchFacetProvider'
 
 
 export default function FacetsPanel() {
-  const searchContext = useContext(StudySearchContext)
+  const searchFacetContext = useContext(SearchFacetContext)
```

Comment:
To enable tests, we should use constructs like `useContextSearchFacet` instead of directly calling `useContext(SearchFacetContext)`.